server/data/data-jmap/src/test/java/org/apache/james/jmap/api/filtering/RuleTest.java

```diff
@@ -103,7 +103,43 @@ void nameShouldBeMandatory() {
                 .condition(CONDITION)
                 .action(ACTION)
                 .build())
-            .isInstanceOf(IllegalStateException.class);
+            .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    void shouldWhenNameEmpty() {
```

Comment:
```suggestion
    void shouldFailWhenNameEmpty() {
```
Same remark for tests below